backend/battling/urls.py

```diff
@@ -1,16 +1,11 @@
 from django.urls import path
 
-from battling.views import BattleEnd, CreateBattle, EnterBattle, Home, Invite, Opponent
-
-from . import views
+from battling.views import CreateBattle, DetailBattle, EnterBattle, Home
 
 
 urlpatterns = [
     path("", Home.as_view(), name="home"),
-    path("create_battle", CreateBattle.as_view(), name="create_battle"),
```

Comment:
Hey, I think you can keep the old ones so we don't break all the site at once. During the refactoring, you can delete old URLs.

Reply:
It's not breaking, because the invite, the opponent, and the end_battle URLs were just to display a message. I changed the route so that it would redirect back to home instead of invite and opponent. You think it's better to still keep them?

Reply:
ok, no problem